src/js/common/model/veteran.js

```diff
@@ -775,13 +775,18 @@ function veteranToApplication(veteran) {
       case 'cityOfBirth':
       case 'stateOfBirth':
       case 'email':
+        if (value.value === '') {
+          return undefined;
+        }
+        break;
+
       case 'homePhone':
       case 'mobilePhone':
       case 'spousePhone':
         if (value.value === '') {
           return undefined;
         }
-        break;
+        return value.value.replace(/[- )(]/g, '');
```

Comment:
Right, but someone could do like `[555] 555-0101` and the `[555]` would get through with this regex, causing the schema validation to fail. I think we're probably fine letting them enter the number how they want, but then stripping it down to what we actually need.